sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -551,7 +562,8 @@ static int _create_candidate_set(gnrc_ipv6_netif_t *iface, const ipv6_addr_t *ds
         /* "In any case, multicast addresses and the unspecified address MUST NOT
          *  be included in a candidate set."
          */
-        if (ipv6_addr_is_multicast(&(iter->addr)) ||
+        if ((iter->flags & GNRC_IPV6_NETIF_ADDR_FLAGS_TENTATIVE) ||
```

Comment:
Well a tentative address is not really assigned to the interface until the registration is complete (see https://tools.ietf.org/html/rfc4862), so it needs to be skipped here... speaking of… I forgot to check for it on receival.